### DISCUSSION TITLE
Can I return my Degen coin?

### DISCUSSION THREAD

**Discussion by buddhaman1** [blockscout/blockscout#9857] [Support]
https://explorer.degen.tips/tx/0x8e34116789fb126b41d6b6794b87f89bac4e04ded3f207cbcdf80573741a9d46

I sent the wrong chain.
Can you send it back to the FROM address?

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
